dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/imports/validation/TrackedEntityImportValidationTest.java

```diff
@@ -126,15 +125,9 @@ void testUpdateAccessInSearchScopeOu() throws IOException {
     ImportReport importReport = trackerImportService.importTracker(params, trackerObjects);
     assertNoErrors(importReport);
     assertEquals(3, importReport.getStats().getCreated());
-    // For some reason teiSearchOrgunits is not created properly from
-    // metadata
-    // Redoing the update here for the time being.
-    User user = userService.getUser(USER_8);
-    user.setTeiSearchOrganisationUnits(new HashSet<>(user.getDataViewOrganisationUnits()));
-    userService.updateUser(user);
-    dbmsManager.clearSession();
```

Comment:
@enricocolasante the change to the TrackedEntityMapper above seems to have resolved this setup issue. Does it look ok to you?